lib/simulator_custatevec.h

```diff
@@ -80,23 +98,30 @@ class SimulatorCuStateVec final {
   void ApplyControlledGate(const std::vector<unsigned>& qs,
                            const std::vector<unsigned>& cqs, uint64_t cmask,
                            const fp_type* matrix, State& state) const {
-    std::vector<int32_t> control_bits;
-    control_bits.reserve(cqs.size());
-
-    for (std::size_t i = 0; i < cqs.size(); ++i) {
-      control_bits.push_back((cmask >> i) & 1);
-    }
+    if (qs.size() == 0) {
+      IO::errorf(
+          "error: controlled global phase gate is not implemented %s %d\n",
```

Comment:
Is this intended for this PR, planned for a future PR, or not on the roadmap?

Reply:
It is not on the roadmap.